SmartPlaces.Facilities/samples/Topology/README.md

```diff
@@ -49,3 +49,86 @@ The following secrets must be defined in the KeyVault in order to run this code:
+
+    1. Ingestion Metrics (how many of each type were found in the source graph): 
+        1. Number of Sites
+        2. Number of Buildings
+        3. Number of Twins
+        4. Number of Relationships
+    2. Twin Metrics
+        1. Number of Twin creates succeeded
+        2. Number of Twin creates failed
+        3. Number of Throttled Twin creates
+        4. Number of Twin updates succeeded
+        5. Number of Twin updates failed
+        6. Number of Throttled Twin updates
+        7. Number of Twin updates skipped (due to no differences between source and target)
+    3. Relationship Metrics
+        1. Number of Relationship creates succeeded
+        2. Number of Relationship creates failed
+        3. Number of Throttled Relationship creates
+        4. Number of Relationship updates succeeded
+        5. Number of Relationship updates failed
+        6. Number of Throttled Relationship updates
+        7. Number of Relationship updates skipped (due to no differences between source and target)
+    4. Ontology Mapping Metrics
+        1. Mapping for Input DTMI not found: 
+            - Counts the number of the Incoming DTMIs that do not exist in the Source Ontology file that is loaded. 
+            - Entries here indicate that the topology program assemblies are out of sync with the source system's ontology
+        2. Output Mapping for Input DTMI not found:
+            - Counts the number of the Incoming DTMIs that are not mapped to output DTMIs in the mapping file
+            - Entries here indicate that the ontology mapping from source to target is incomplete and should be updated. It is possible that the target ontology is a smaller set than the
+              input ontology and some mappings may not be possible (or needed).
+        3. Target DTMI not found:
+            - An target dtmi entry exists in the mapping file which does not actually exist in the target ontology
+        4. InputInterfaceNotFound not found in Model by InterfaceType
+            - A source dtmi entry exists in the mapping file which does not actually exist in the source ontology
+        5. Relationship Not Found in Model by RelationshipType
+            - A relationship was found in the toplogy that is not found in the target ontology
+        6. Duplicate property found in Model by PropertyName
+            - A invalid model was found with multiple properties with the same name
+        7. Invalid Target DTMI Mappings in MappingFile
+            - The mapping file has a dtmi that is not in the target ontology
+        8. Invalid Output DTMI by OutputDtmi
+            - The mapping file has a malformed DTMI
+
+3. If you have set up logging for your container instance and routed the logs to a log analytics instance, you can look at the CustomLogs / ContainerInstanceLog_CL in your Log Analytics workspace. If your instance name is "topology", you can run the following query to get all of the logs 
```

Comment:
Do you also want to include that the same results are available in AppInsights?

```
traces
| where customDimensions.CategoryName startswith 'Microsoft.SmartPlaces.Facilities.'
```

```
exceptions
| where customDimensions.CategoryName startswith 'Microsoft.SmartPlaces.Facilities.'
```

Reply:
As far as I know this is the first mention of LogAnalytics in this repo since we've given no guidance on how to deploy the code. We have however given inputs for AppInsights